specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -1652,6 +1652,90 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.SecurityInsights/enrichment/ip/geodata/": {
```

Comment:
This is being added back to 2019-01-01-preview? As readme.md default tag is referring to 2020 stable, the SDK automation generated no diff. You should really create a new API versions for it.

Reply:
This is because this PR is actually a copy of one made a while back (which received ARM approval and approval for the breaking change).    The reason I had to make this fresh PR was that I couldn't convert my PR in the private repo into a PR in the public repo (I had permissions problems, and a 500-level error when I tried). I contacted the OpenAPI support staff and Zhenglai Zhang suggested I make a brand new PR and just reference the old one.

Reply:
@allenjzhang (as above) Can you resolve this and approve/complete the PR?